### PR TITLE
Please update the latest Payara versions on the compatibility page #1203

### LINKED ISSUE #1203: Please update the latest Payara versions on the compatibility page
To include your Compatible Software Product on the Jakarta EE official list of Compatible Software Products, please provide the following information:

**The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting**
https://accounts.eclipse.org/users/lprimak

**A product name and version**
Payara Community 5.2021.3, Payara Enterprise 5.28.0
Currently only 5.22.0 and 5.23.0 are specified under Jakarta 8 Compatibility

**The Profile (Full Platform, Web Profile, Both) that the Compatible Product is submitted to be listed under**
Both Profiles

**A ZIP file with a product logo with a transparent background in SVG format (optional)**
Already exists on the page, just want to update the version number to recent

**The Participant, Enterprise or Strategic Member name supporting the Compatible Software Product**
Payara Foundation

**The URL to the approved certification issue demonstrating compatibility as defined in the Jakarta EE TCK Process**
https://github.com/eclipse-ee4j/jakartaee-platform/issues/311

**The URL where Compatible Software Product is available for download**
https://www.payara.fish/downloads/

Thank you!

### PR DESCRIPTION
**Pull request by autumnfound**
Fixes #1203

### CODE REVIEW

**Comment by lprimak** [comment]
Hi, @autumnfound 
Thank you for looking at this issue.
However, the latest versions are 5.2021.3 (Community) and 5.28.0 (Enterprise)
Can you reflect those on the page please as opposed to the older versions in this PR?

Thank you!

File: data/compatible_products.yml [hunk 1]
```diff
@@ -198,6 +198,9 @@ sets:
         - version: 5.23.0
           compatibility: 'https://docs.payara.fish/enterprise/docs/5.23.0/jakartaee-certification/5.23.0/tck-results-full-5.23.0.html'
           download_url: 'https://docs.payara.fish/enterprise/docs/5.23.0/jakartaee-certification/5.23.0/tck-results-full-5.23.0.html'
+        - version: 5.26.0
```
Comment (lprimak) [inline data/compatible_products.yml]: Should be 5.28.1

File: data/compatible_products.yml [hunk 2]
```diff
@@ -399,3 +402,6 @@ sets:
         - version: 5.23.0
           compatibility: 'https://docs.payara.fish/enterprise/docs/5.23.0/jakartaee-certification/5.23.0/tck-results-web-5.23.0.html'
           download_url: 'https://docs.payara.fish/enterprise/docs/5.23.0/jakartaee-certification/5.23.0/tck-results-web-5.23.0.html'
+        - version: 5.26.0
```
Comment (lprimak) [inline data/compatible_products.yml]: Should be 5.28.1

**Review by lprimak — CHANGES_REQUESTED** [review]
Please add Community 5.2021.4 as a compatible version as well,
https://www.payara.fish/downloads/payara-platform-community-edition/
thank you!

**Comment by autumnfound** [comment]
@lprimak your comments reflect new versions that are not yet validated as certified, and is different than the versions originally requested. We can convert this to draft and wait until the new certification request is finalized, and then update the links accordingly. Does that work with you?

**Comment by lprimak** [comment]
@autumnfound Yes, Draft sounds good.
See https://github.com/eclipse-ee4j/jakartaee-platform/issues/369
The next versions to be certified would be 5.28.1 and 5.2021.4

Thank you!

**Comment by lprimak** [comment]
Hi, @autumnfound 
I *believe* the above releases are now Jakarta EE certified

Thank you!

**Comment by autumnfound** [comment]
@ivargrimstad can you confirm? Its not closed but its got the accepted tag

**Comment by autumnfound** [comment]
@ivargrimstad @TanjaObradovic So this update would push 2 different product streams under the same product. How do we want to handle this in the UI? 2 separate entries for Payara server, or 2 versions under the same product? I've gone the second for now as it is the most conservative option, and I'm not sure what the procedure is here.

**Comment by ivargrimstad** [comment]
> @ivargrimstad @TanjaObradovic So this update would push 2 different product streams under the same product. How do we want to handle this in the UI? 2 separate entries for Payara server, or 2 versions under the same product? I've gone the second for now as it is the most conservative option, and I'm not sure what the procedure is here.

I think we should ask Payara for input on whether they see the Payara Server Enterprise and Payara Server Community editions as different versions of the same product, or as distinct products. 
If they are different products, they should be listed separately, similar to e.g. JBoss AS and WildFly. The only difference here is that the products have the same logo, but different names/versions.

**Comment by lprimak** [comment]
@ivargrimstad @TanjaObradovic @autumnfound 
Hi,
We had an internal discussion here at Payara, and we agree that we want Payara Enterprise and Community to be separate products on the Eclipse Compatibility page.

Can we make that happen? Thank you so much!

**Comment by lprimak** [comment]
@autumnfound I would also remove the older versions of Payara from the page, I don't believe there is a need to keep them. Just the latest versions.

**Review by lprimak — APPROVED** [review]

**Comment by lprimak** [comment]
Perfect! Thank you @autumnfound

**Review by chrisguindon — APPROVED** [review]

**Comment by TanjaObradovic** [comment]
having Payara Enterprise and Community to be separate products on the Eclipse Compatibility page, makes perfect sense to me.

@autumnfound please let us know if you need any other details regarding this.

**Pull request merged**